code-challenges.js

```diff
@@ -10,7 +10,48 @@ var people = [
 ]
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is a president of the galaxy." "Arthus Dent is a radio employee."
 
+const sent = (obj) => {
```

Comment:
parameter naming of `obj` is confusing because `obj` is usually used for objects which have key value pairs

Reply:
also a more descriptive function name would be helpful. `sent` doesn't quite tell me what it's doing